bin/mutatex

```diff
@@ -301,23 +343,22 @@ except:
 
 residues_list = []
 dir_list = []
+
 for pdb in repaired_pdbs_list:
     this_dirname = working_directory+"/"+os.path.splitext(os.path.basename(pdb))[0]
-    for pdb in repaired_pdbs_list:
-        this_dirname = working_directory+"/"+os.path.splitext(os.path.basename(pdb))[0]
```

Comment:
thanks for spotting this! This nested for is just weird and probably useless, but it shouldn't have ill effects (except maybe some extra lines in the terminal output)